package.json

```diff
@@ -133,6 +133,7 @@
     "import": "0.0.6",
     "intercom-client": "^2.10.2",
     "intl": "^1.2.4",
+    "joplin-turndown-plugin-gfm": "^1.0.12",
```

Comment:
Why `joplin-turndown-plugin-gfm` rather than `turndown-plugin-gfm`? Activity on that fork is low enough to feel like a red flag to me.

Reply:
Last commit on `turndown-plugin-gfm` was in 2018. Last commit on the joplin one was 2020, fixing a number of bugs that seem to be ignored on the main repo.